quill-core/src/test/scala/io/getquill/quotation/QuotationSpec.scala

```diff
@@ -832,8 +834,8 @@ class QuotationSpec extends Spec {
         l.encoder mustEqual intEncoder
       }
       "property" in {
-        case class Test(a: String)
-        val t = Test("a")
+        case class TestEntity(a: String)
+        val t = TestEntity("a")
```

Comment:
I forgot detailed error however it is connected with suspicious shadowing of generated class name or something like that